### ISSUE TITLE
Snapshot black when not streaming on USB Device

### ISSUE THREAD

**Issue by cavernerg** [AlexxIT/go2rtc#1015]
I use a logitech USB Cam to monitor my 3D printer. When I grab a snapshot (via Octoslack/Octoprint) I get a black image. If I stream while Octosclack downloads the snapshot, it's fine.

**Comment by cavernerg** [comment]
Thanks for replying. Here is an example picture
![WhatsApp Bild 2024-04-30 um 20 15 12_008f86b1](https://github.com/AlexxIT/go2rtc/assets/62744005/d236c181-b77f-45d0-bc22-1f61589c78b3)

This is the entry in config file: 
ffmpeg:device?video=1#video=mjpeg#hardware

**Comment by AlexxIT** [comment]
Show what you can see WebUI > Add > USB

**Comment by cavernerg** [comment]
![image](https://github.com/AlexxIT/go2rtc/assets/62744005/30478cf5-65c5-4a5d-ab99-5581d5af685d)

it is a Logitech C615 Camera. To me it seems like the camera is in standby and when go2rtc takes  the snapshot, the camera is not fully awake and adjusted to the light. 
Because if I am watching the stream on my android and go2rtc takes the snapshot in the same moment, everything is fine.

**Comment by AlexxIT** [comment]
You config as suggested in WebUI. You don't need transcoding for mjpeg, because camera support this format out of box.

**Comment by cavernerg** [comment]
Oh right, I was there before and tried around - but the result remains the same.

**Comment by AlexxIT** [comment]
What about stream? Do you have problems only with snapshots?

**Comment by cavernerg** [comment]
Stream works flawless. Only snapshots are problematic. And that only, when camera is not in use e.g. through parallel streaming.

**Comment by AlexxIT** [comment]
Have you tried to get snapshot during active stream?

**Comment by cavernerg** [comment]
Yeah that is what I tried to explain. In this moments , snapshots are fine

**Comment by AlexxIT** [comment]
You have two examples in the go2rtc WebUI. Have you tried them both?

**Comment by cavernerg** [comment]
Yes , I did.
To me it seems like the camera needs too much time to wake up from standby

**Comment by cavernerg** [comment]
Yeah I'd do that , if it was my own implementation of getting the snap, but I'm using octoprint/octoslack to do that. :(
